### PR TITLE
Delete and Update Store

### PR DESCRIPTION
**Pull request by Thobeats**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: app/Http/Controllers/StoreController.php [hunk 1]
```diff
@@ -212,6 +249,29 @@ public function update(Request $request, $id)
...
+
+            if($status == 200){
+                // $data = [
+                //     "message" => "Store Deleted",
+                //     "class" => "alert-success"
+                // ];
+
+                return view('backend.stores.index')->with('data', "Store Deleted");
+            }
+
```
Comment (Jeremiahiro) [inline app/Http/Controllers/StoreController.php:271]: so what happens if the status is not equal to 400?

File: resources/views/backend/stores/edit.blade.php [hunk 2]
```diff
@@ -74,7 +88,7 @@
                 </div>
                 <div class="modal-footer">
                   <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                  <button type="button" class="btn btn-success"><a href="/admin/view_store" class="text-white">Save changes</a></button>
+                  <button type="submit" class="btn btn-success"><a href="/admin/view_store" class="text-white">Save changes</a></button>
```
Comment (Jeremiahiro) [inline resources/views/backend/stores/edit.blade.php:91]: this route does not exist, if you created it, you'll have to delete and use whats appropriate

**Comment by Jeremiahiro** [comment]
Please review

**Comment by Thobeats** [comment]
I didn't create the route, I met it there

**Pull request closed without merging**